<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>МС6205 - control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="TextEditScreen_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>16</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Mono</family>
      <pointsize>17</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEditScreen_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>16</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Mono</family>
      <pointsize>17</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEditScreen_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>16</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Mono</family>
      <pointsize>17</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEditScreen_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>16</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Mono</family>
      <pointsize>17</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTake_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Screen 1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTake_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Screen 2</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTake_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Screen 3</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTake_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Screen 4</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearScreen_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сlear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearScreen_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сlear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearScreen_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сlear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearScreen_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сlear</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear all screens</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoUpdate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SecondsAutoUpdate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxClock">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
